src/components/ebay-combobox/component.ts

```diff
@@ -35,6 +36,7 @@ interface ComboboxInput extends Omit<Marko.HTML.Input, `on${string}`> {
         }>;
     options?: Marko.AttrTag<ComboboxOption>;
     "chevron-size"?: "large";
+    "dropdown-element"?:  () => HTMLElement;
```

Comment:
Initially it was not clear to me what this was supposed to be. I think `"dropdown-source"` might make more sense as a name.

Also, please add to `.stories.ts`, I looked for a description and couldn't find one 😢

Reply:
This is simply an internal api. No need to document it.

Reply:
Ah okay, if it's internal only can you add a JSDoc comment for it? Something like `/** For internal use only */`

That said, are there any use cases where teams have custom comboboxes and might need to use this on their own?

Reply:
I believe this is only needed when combobox styles are overridden such as in chips combobox.